### PR TITLE
fix: Feedback Addressed by @Marcony1 about adding repostatus.org badg…

### LINKED ISSUE #89: Add repostatus.org badge
Need to ad a badge from https://www.repostatus.org as mandated by the review template.

Also pointed out by reviewers in external issue https://github.com/UBC-MDS/software-review-2024/issues/18.

### PR DESCRIPTION
**Pull request by Marcony1**
…e - suggested by @wqxxzd @riyaeliza123 and @charlesxch

### CODE REVIEW

**Comment by SoloSynth1** [comment]
This closes #89.

File: README.md [hunk 1]
```diff
@@ -4,9 +4,13 @@
 
 [![GitHub Release](https://img.shields.io/github/release/ubc-mds/visumorph.svg?style=flat)]()
 [![GitHub Activity](https://img.shields.io/github/last-commit/ubc-mds/visumorph/main.svg?style=flat)]()
+[![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
 [![Documentation Status](https://readthedocs.org/projects/visumorph/badge/?version=latest)](https://visumorph.readthedocs.io/en/latest/?badge=latest)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 [![codecov](https://codecov.io/github/UBC-MDS/VisuMorph/graph/badge.svg?token=rnxgkcvwda)](https://codecov.io/github/UBC-MDS/VisuMorph)
+.. image:: https://www.repostatus.org/badges/latest/active.svg
+   :alt: Project Status: Active – The project has reached a stable, usable state and is being actively developed.
+   :target: https://www.repostatus.org/#active
```
Comment (SoloSynth1) [inline README.md]: I think you added these three lines by mistake?

Comment (Marcony1) [reply README.md]: Oh, you are right. Added them, and adapted them, but forgot to delete them. Allow me

Comment (SoloSynth1) [reply README.md]: no worries!

Comment (Marcony1) [reply README.md]: Addressed

Comment (SoloSynth1) [reply README.md]: Resolved in [9c49585](https://github.com/UBC-MDS/VisuMorph/pull/99/commits/9c495850f801a983e5a0576b35233f20a32cc839).

**Review by SoloSynth1 — APPROVED** [review]
LGTM!

**Comment by SoloSynth1** [comment]
@Marcony1 Thanks for the PR! Merging to `main`.

**Pull request merged**